run_ortmodule_mvp_bert_finetuning.sh

```diff
@@ -1,19 +0,0 @@
-#!/bin/bash
-
-cur_dir=$(basename `pwd`)
-
-if [[ ${cur_dir} != "RelWithDebInfo" ]]
-then
-    echo "Going to build folder (aka build/Linux/RelWithDebInfo)"
-    cd build/Linux/RelWithDebInfo
-fi
-
-echo "Exporting PYTHONPATH to use build dir as onnxruntime package"
-export PYTHONPATH=$(pwd)
-
-echo "Copying PyTorch frontend source-code to build folder"
-cp -Rf ../../../orttraining/orttraining/python/training/* ../../../build/Linux/RelWithDebInfo/onnxruntime/training/
-
-echo "Running Flexible API (ORTModule)"
-python ../../../orttraining/orttraining/test/python/orttraining_test_ortmodule_bert_classifier.py --help
```

Comment:
Thanks for cleaning this up. 
IIUC, these are part of the ortmodule_tests.py now ?

Reply:
Hmm... It might worth making a simple convergence tests out of these simple models. 
I will add a work item for this.

Reply:
https://msdata.visualstudio.com/Vienna/_sprints/taskboard/ONNX%20Training/Vienna/Cobalt?workitem=1069346